__fixtures__/test-project/web/package.json

```diff
@@ -17,12 +17,13 @@
     "@redwoodjs/forms": "3.0.2",
     "@redwoodjs/router": "3.0.2",
     "@redwoodjs/web": "3.0.2",
+    "humanize-string": "2.1.0",
```

Comment:
Hey all, saw this change when merging the patch release branch back into main. Could we go without this change, or is this package necessary for this PR? I noticed we didn't change the CRWA template, so something may be missing of this one's necessary.

Reply:
Interesting that it shows up here now.
Danny and I discovered a while ago that it's been missing as a web side dependency. 
Let me try to find a PR that's relevant

Reply:
Here's the PR
https://github.com/redwoodjs/redwood/pull/6221/files

humanize-string should not be a dep in the test project